src/main/java/org/broadinstitute/hellbender/utils/IntervalUtils.java

```diff
@@ -137,7 +153,20 @@ else if (isIntervalFile(arg)) {
                 throw new UserException.MalformedFile(new File(arg), "Interval file could not be parsed in any supported format.", e);
             }
         }
-        // otherwise treat as an interval -> parse and add to raw interval list
+        // If it's neither a Feature-containing file nor an interval file, but is an existing file, throw an error
+        else if ( new File(arg).exists() ) {
+            throw new UserException.CouldNotReadInputFile(arg, String.format("The file %s exists, but does not contain Features " +
+                    "(ie., is not in a supported Feature file format such as vcf, bcf, or bed), " +
+                    "and does not have one of the supported interval file extensions (" + INTERVAL_FILE_EXTENSIONS + "). " +
+                    "Please rename your file with the appropriate extension. If %s is NOT supposed to be a file, " +
+                    "please move or rename the file at location %s", arg, arg, new File(arg).getAbsolutePath()));
+        }
+        // If it's not an existing file, but looks like it was supposed to be a file (ie., it has an extension), throw an error.
+        // This works because '.' is not an allowable character in interval strings.
+        else if ( arg.indexOf('.') != -1 ) {
```

Comment:
Slightly more efficient when searching for individual characters, since `String.contains()` takes a `CharSequence` and calls `toString()` on its argument before invoking `indexOf(String)`